content/integrate/redis-data-integration/_index.md

```diff
@@ -26,40 +25,89 @@ RDI's purpose is to help Redis customers sync Redis Enterprise with live data fr
+- **Easy to install and operate** - Use a self documenting command line interface (CLI)
+  for all installation and day-two operations.
+- **No coding needed** - Create and test your pipelines using Redis Insight.
+- **Data-in-transit encryption** - RDI never persists data to disk. All data in-flight is
+  protected using TLS or mTLS connections.
+- **Observability - Metrics** - RDI collects data processing counters at source table granularity
+  along with data processing performance metrics. These are available via GUI, CLI and 
+  [Prometheus](https://prometheus.io/) endpoints.
+- **Observability - logs** - RDI saves rotating logs to a single folder. They are in a JSON format,
+  so you can collect and process them with your favorite observability tool.
+- **Backpressure mechanism** - RDI is designed to backoff writing data when the cache gets
+  disconnected, which prevents cascading failure. Since the change data is persisted in the source
+  database and Redis is very fast, RDI can easily catch up with missed changes after a short period of
+  disconnection. See [Backpressure mechanism]({{< relref "/integrate/redis-data-integration/architecture #backpressure-mechanism">}}) for more information.
+- **Recovering from full failure** - If the cache fails or gets disconnected for a long time,
+  RDI can reconstruct the cache data in Redis using a full snapshot of the defined dataset.
+- **High throughput** - Because RDI uses Redis for staging and writes to Redis as a target,
+  it has very high throughput. With a single processor core and records of about 1KB in size,
+  RDI processes around 10,000 records per second. While taking the initial full *snapshot* of
+  the source database, RDI automatically scales to a configurable number of processing units,
+  to fill the cache as fast as possible.
+
+## When to use RDI
+
+RDI is designed to support apps that must use a disk based database as the system of record
+but must also be fast and scalable. This is a common requirement for mobile and web
+apps with a rapidly-growing number of users; the performance of the main database is fine at first
+but it will soon struggle to handle the increasing demand without a cache.
+
+You should use RDI when:
+
+- You must use a slow database as the system of record for the app 
+- The app must always *write* its data to the slow database
+- You already intend to use Redis for the app cache
+- The data changes frequently in small increments
+- Your app can tolerate *eventual* consistency of data in the Redis cache
+
+You should *not* use RDI when:
+
+- You are migrating an existing data set into Redis only once
+- The data is updated infrequently and in big batches
+- Your app needs *immediate* cache consistency rather than *eventual* consistency
+- The data is ingested from two replicas of Active-Active at the same time
+- The app must *write* data to the Redis cache, which then updates the source database
+- Your data set will only ever be small
```

Comment:
```suggestion
- You are migrating an existing data set into Redis only once.
- The data is updated infrequently and in big batches.
- Your app needs *immediate* cache consistency rather than *eventual* consistency.
- The data is ingested from two replicas of Active-Active at the same time.
- The app must *write* data to the Redis cache, which then updates the source database.
- Your data set will only ever be small.
```